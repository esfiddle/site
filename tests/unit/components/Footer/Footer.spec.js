import { shallowMount } from "@vue/test-utils";
import Footer from "@/components/Footer/Footer.vue";

describe("Footer.vue", () => {
  it("renders Footer properly", () => {
    const wrapper = shallowMount(Footer);
    expect(wrapper).toMatchSnapshot();
  });
});
