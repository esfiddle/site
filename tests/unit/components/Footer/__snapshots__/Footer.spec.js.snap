// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue renders Footer properly 1`] = `
<footer>
  <router-link to="/">
    <!---->
  </router-link>
  <div class="spacer"></div> <a href="https://www.netlify.com"><img src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg"></a>
  <div class="spacer"></div>
  <div class="footerLinks">
    <router-link to="/examples">Examples</router-link>
    <router-link to="/editor">Editor</router-link>
    <router-link to="/about">About</router-link>
    <router-link to="/blog">Blog</router-link>
  </div>
</footer>
`;

exports[`Footer.vue renders props.msg when passed 1`] = `
<footer>
  <router-link to="/">
    <!---->
  </router-link>
  <div class="spacer"></div> <a href="https://www.netlify.com"><img src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg"></a>
  <div class="spacer"></div>
  <div class="footerLinks">
    <router-link to="/examples">Examples</router-link>
    <router-link to="/editor">Editor</router-link>
    <router-link to="/about">About</router-link>
    <router-link to="/blog">Blog</router-link>
  </div>
</footer>
`;
